styleguide/derek/templates/index.jade

```diff
@@ -4,13 +4,15 @@
 
 .row.half-padding-top
   p Below is a list of examples to help you get started building your web pages.
+  p
+    a(href="https://raxglobal.sharepoint.com/Global%20Ecommerce/Shared%20Documents/Forms/AllItems.aspx?id=%2FGlobal%20Ecommerce%2FShared%20Documents%2FeComm%20Ops%2FWeb%20page%20changes%2FTemplate%20Documentation") How to write to our Templates
```

Comment:
Let's mask this URL. You can use https://rax.io/create to create shortened/obfuscated URLs that only resolve in-network.